reference/shipping.v3.yml

```diff
@@ -142,6 +143,61 @@ paths:
                     CA: '508313'
                     AU: '817355'
                     ALL: '501000'
+
+  '/shipping/settings':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+
+    get:
+      operationId: getShippingSettings
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/shippingSettings'
+                  meta:
+                    $ref: '#/components/schemas/metaCollection'
+      description: |-
+        Get shipping settings.
+      tags:
+        - Shipping Settings
+      summary: Get Shipping Settings
+    post:
+      operationId: updateShippingSettings
+      parameters:
+        - $ref: '#/components/parameters/ContentType'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
```

Comment:
As also shown in @bc-andreadao's screenshot - this isn't an array of shipping settings, this is just an object. Providing the actual request and response payloads should highlight this inconsistency.

It looks like this is a copy paste from customs information, which is indeed an array of various customs information entities.

Reply:
Thanks Brett, yeah right my bad - I've updated the type to object. Gotta scan better next time when I copy paste from some other existing code😅